tests/unit/constraints/test_tabular.py

```diff
@@ -875,7 +877,7 @@ def test_is_valid_strict_true(self):
           for the other two.
         """
         # Setup
-        instance = GreaterThan(low='a', high='b', strict=True)
+        instance = GreaterThan(low='a', high='b', strict=False)
```

Comment:
I think this change and the ones below sapping True/False should not be here, since now they do not match the test_case names

Reply:
I forgot to swap the test names. I changed `strict` to `False` to match the documentation.

Reply:
Oh! I did not realize that the docs did not match the method name. Good catch!